### ISSUE TITLE
MRO Control Zero H7 Support - Handling USART6 vs RC

### ISSUE THREAD

**Issue by N0L12** [PX4/PX4-Autopilot#18403]
I am programming a driver in PX4 to be run on a mro-ctrl-zero-h7 board, in order to manage a serial sensor attached on USART6 (pin 8 and 10 of rear connector).

According to the documentation USART6 should be mapped on /dev/ttyS3. Unfortunately it seems like the input is overlapped with the RC input, which is by default also set to use the /dev/ttyS3 as defined in the "/boards/mro/ctrl-zero-h7/src/board_config.h" (#define RC_SERIAL_PORT "/dev/ttyS3"). The RC is physically connected to the pin 3 of the FRONT connector and it is correctly working with this default setting, with a PPM signal.

In conclusion I am not sure about how to correctly handle the USART6 of MRO Control Zero H7 in PX4 without conflicting with the RC.
Issue https://github.com/PX4/PX4-Autopilot/issues/16569 seems not solving the problem.

FYI, the same board with Ardupilot is correctly handling all the devices...

**Comment by N0L12** [comment]
Yes, I want to use the RC input connected to the pin 3 of the FRONT connector together with another external sensor connected to the UART6 (pin 8 and 10 of rear connector).
But it seems like even if they are physically separated, both connections are mapped as "/dev/ttyS3" and I am confused how to handle them separately...
